perun-core/src/main/java/cz/metacentrum/perun/core/bl/SecurityTeamsManagerBl.java

```diff
@@ -99,8 +99,31 @@
 	 * @param securityTeam
 	 * @return list of users which are security admins in security team
 	 * @throws InternalErrorException
+         * @@throws SecurityTeamNotExistsException
 	 */
-	List<User> getAdmins(PerunSession sess, SecurityTeam securityTeam) throws InternalErrorException;
+	List<User> getAdmins(PerunSession sess, SecurityTeam securityTeam, boolean onlyDirectAdmins) throws InternalErrorException, SecurityTeamNotExistsException;
+
+	/**
+         * Gets list of direct user administrators of the SecurityTeam.
+         * 'Direct' means, there aren't included users, who are members of group administrators, in the returned list.
+         *
+         * @param perunSession
+         * @param securityTeam
+         * @return list of users which are security admins in security team
+         * @throws InternalErrorException
+         */
+        @Deprecated
+        List<User> getDirectAdmins(PerunSession perunSession, SecurityTeam securityTeam) throws InternalErrorException;
```

Comment:
I don't think is good to create "deprecated" method from start. I think that getAdmins with "onlyDirectAdmins" parameter is enough.